pkgs/universal_storage_filesystem/Makefile

```
@@ -0,0 +1,4 @@
+gen-rewrite:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)